src/tooltip.js

```diff
@@ -98,7 +102,12 @@ nv.models.tooltip = function() {
 
         trowEnter.append("td")
             .classed("value",true)
-            .html(function(p, i) { return valueFormatter(p.value, i, p) });
+            .html(function(p, i) {
+                if (enableComplexValueFormatter) { // Mirror Chart
+                    return valueFormatter(p, i);
+                }
+                return valueFormatter(p.value, i, p);
```

Comment:
Oh... I may not understand something here...

If `enableComplexValueFormatter === true`, we pass `p` (point) as the first attribute to `valueFormatter` which returns `value` property of passed point.

If `enableComplexValueFormatter === false`, we pass `p.value` as the first attribute to `valueFormatter` which just returns a passed argument, which is what we passed – `value` property of the point.

... So, what's `enableComplexValueFormatter` for?

And also why `valueFormatter` has all other arguments (`i` and `p`) if they are not used? 

Probably, I missing something, and maybe some of the methods can be overridden and then it all will make sense. But could you please clarify it for me?